### PR TITLE
Metrics and events sweep

### PR DESCRIPTION
**Pull request by homelessbirds**

### CODE REVIEW

File: src/content/docs/apm/agents/php-agent/php-agent-api/newreliccustommetric-php-agent-api.mdx [hunk 1]
```diff
@@ -31,7 +31,7 @@ Record [custom metrics](/docs/agents/manage-apm-agents/agent-data/custom-metrics
 
 Then you can:
 
-* Use the [data explorer](/docs/query-your-data/explore-query-data/data-explorer/introduction-data-explorer) to search for your custom metrics, create customizable charts, and add them to dashboards.
+* Use [metric and events](/docs/query-your-data/explore-query-data/data-explorer/introduction-data-explorer) to search for your custom metrics, create customizable charts, and add them to dashboards.
```
Comment (jeff-colucci) [inline src/content/docs/apm/agents/php-agent/php-agent-api/newreliccustommetric-php-agent-api.mdx:34]: Singular metric

File: src/content/docs/apm/agents/python-agent/python-agent-api/recordcustommetrics-python-agent-api.mdx [hunk 2]
```diff
@@ -93,4 +93,4 @@ newrelic.agent.record_custom_metrics(metrics())
...
+To view custom metrics, use [metric and events](/docs/query-your-data/explore-query-data/data-explorer/introduction-data-explorer) to search and filter for custom metrics, create customizable charts, and add those charts to New Relic dashboards. You can use our [REST API](/docs/apis/rest-api-v2/requirements/new-relic-rest-api-v2-getting-started) to programmatically retrieve and use custom metric data outside of the UI. It is also possible to [create custom metric alert conditions](/docs/alerts/new-relic-alerts/configuring-alert-policies/define-custom-metrics-alert-condition) to notify you or your team when your custom metric exceeds specific values.
```
Comment (jeff-colucci) [inline src/content/docs/apm/agents/python-agent/python-agent-api/recordcustommetrics-python-agent-api.mdx:96]: Singular metric

File: src/content/docs/infrastructure/amazon-integrations/get-started/introduction-aws-integrations.mdx [hunk 3]
```diff
@@ -55,7 +55,7 @@ Keep in mind the following items:
 
 ## View your AWS data [#insights]
 
-Once you follow the configuration process, data from your Amazon Web Services report directly to New Relic. AWS entities for most used services will be listed in the [New Relic Explorer](/docs/new-relic-one/use-new-relic-one/core-concepts/new-relic-explorer-view-performance-across-apps-services-hosts/). Metrics and events will appear in the [data explorer](/docs/query-your-data/explore-query-data/browse-data/introduction-data-explorer/). AWS data will also be visible in the Infrastructure UI.
+Once you follow the configuration process, data from your Amazon Web Services report directly to New Relic. AWS entities for most used services will be listed in the [New Relic Explorer](/docs/new-relic-one/use-new-relic-one/core-concepts/new-relic-explorer-view-performance-across-apps-services-hosts/). Metrics and events will appear in [metrics and explorer](/docs/query-your-data/explore-query-data/browse-data/introduction-data-explorer/). AWS data will also be visible in the Infrastructure UI.
```
Comment (jeff-colucci) [inline src/content/docs/infrastructure/amazon-integrations/get-started/introduction-aws-integrations.mdx:58]: Missing events

**Review by jeff-colucci — DISMISSED** [review]
Just a few minor things, other than those 3 it looks good!

**Review by jeff-colucci — APPROVED** [review]
Looks good!

**Comment by homelessbirds** [comment]
merging with permission from @LizBaker !

**Pull request merged**